cosmos/azure-spring-data-cosmos/cosmos-aad-sample/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>cosmos-spring-aad-sample</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <properties>
+    <spring.boot.version>2.7.1</spring.boot.version>
+    <azure.spring.data.cosmos.version>3.29.1</azure.spring.data.cosmos.version>
+  </properties>
+  <parent>
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../pom.xml</relativePath>
+  </parent>
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-data-cosmos</artifactId>
+      <version>${azure.spring.data.cosmos.version}</version>
```

Comment:
Delete this, because the version is already managed in `spring-cloud-azure-dependencies`, and the parent (`azure-spring-boot-samples`) already imported `spring-cloud-azure-dependencies`.

1. Please delete for all other poms in current PR.
2. Please update in 4.4.1 branch.

Reply:
Deleted version. This was only present in this sample.